docs/securelaunch/README.md

```diff
@@ -0,0 +1,120 @@
+**Note**: Policy file is a *JSON* file, and there is no support for comments in a JSON file.
+***
+
+"default_action": ""
+=========
+Only "" default action supported at this point.
+
+"collectors":
+=========
+Only four collectors supported at this point. They are dmi, file, storage and cpuid.
+## dmi collector:
+
+measures output of dmidecode based on input
+type. input type (like BIOS, processor, system etc) 
+can be provided in policy file as shown.
+```
+  {
+      "type": "dmi",
+      "events": [
+           {
+             "label": "BIOS",
+             "fields": []
+           },
+           {
+             "label": "System",
+             "fields": []
+           },
+           {
+             "label": "Processor",
+             "fields": []
+           }
+  }
+```
+
+## file collector: 
+
+measures any file that platform owner
+wants to measure. target file to measure is input in the
+paths field in policy file as shown
+```
+  {
+     "type": "files",
+     "paths": [ "sda1:/Paul/foo" ]
+  },
+```
+
+## storage collector:
+
+measures an entire disk. target disk to measure
+can be provided in the policy file as shown
+```
+  {
+      "type": "storage",
+      "paths": [ "/dev/sdb1" ]
+  },
+```
+
+## cpuid collector:
+
+measures (hashes) cpuid data of the platform
+and stores the result on a file. target file is input as
+location field in policy file.
+e.g
+```
+  {
+     "type": "cpuid",
+     "location": "sda1:/Paul/foo"
+  }
+```
+
+"attestor": {}
+=========
+a nil slice is only supported at this point.
+
+"launcher":
+=========
+launcher provides ability to kexec into a target kernel.
+platform owners can choose which target kernel to kexec into
+by setting launcher module in the platform policy file
+as shown.
+```
+"launcher": {
+   "type": "kexec",
+   "params": {
+   "initrd":"sdc1:/boot/initramfs-4.14.35-1941.el7uek.x86_64.img",
```

Comment:
can you "format" the json in this file? consistent indentation etc

Reply:
Hmm the idea was to keep the indentation identical to the original policy file so that platform admins who have no experience with this file can compare the two...but I can indent them all equally..just afraid it might confuse them...